addon/components/polaris-drop-zone/file-upload.js

```diff
@@ -53,6 +46,16 @@ export default Component.extend({
     return fileUpload[`actionHint${classify(type)}`];
   }),
 
+  iconDragDrop,
+
+  assetFileUpload,
+
+  assetImageUpload,
+
+  type: readOnly('context.type'),
```

Comment:
these seems to have been moved after multi-lines...any idea why?

Reply:
because those are public while these are private